101-1vm-2nics-2subnets-1vnet/azuredeploy.json

```diff
@@ -124,12 +124,25 @@
       "kind": "Storage",
       "properties": {}
     },
+    {
+      "comments": "Simple Network Security Group for subnet [variables('subnet2Name')]",
+      "type": "Microsoft.Network/networkSecurityGroups",
+      "apiVersion": "2019-08-01",
+      "name": "[concat(variables('subnet2Name'), '-nsg')]",
+      "location": "[parameters('location')]",
+      "properties": {
+        "securityRules": []
```

Comment:
remove this since it's empty - just the securityRules property, if it's require lmk and I'll add a test exception for it

Reply:
Removed.